### PR TITLE
Marcos.reginato/funtionality c mult buttons

### PR DESCRIPTION
**Pull request by regmarcos**
Hi! I just added the clear button funcionality, now you can erase the last symbol that you have pressed, it can be a number or an operator. If you do a long press in the clear button it will erase everything in the visor.

capturaPR2

Added a clearAll button and fixed some mistakes in relation with the operators

### CODE REVIEW

File: app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java [hunk 1]
```diff
@@ -4,16 +4,18 @@
 import android.os.Bundle;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
+import butterknife.OnFocusChange;
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java]: remove the **OnFocusChange** you don't use it now

File: app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java [hunk 2]
```diff
@@ -102,4 +104,19 @@ public void onButtonSumPressed(){
...
+        presenter.onOperatorPressed(MULTIPLICATION);
+    }
+
+    @OnClick(R.id.button_clear)
+    public void onButtonClearPressed(){
+        presenter.onClearPressed();
+    }
+
+    @OnLongClick(R.id.button_clear)
+    public void onButtonClearLongClicked(){
```
Comment (SartiMau) [inline app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java]: remove this as you do not use it any more

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java (hunk 2 above)
Comment (regmarcos) [reply app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java]: done

File: app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java (hunk 1 above)
Comment (regmarcos) [reply app/src/main/java/com/example/calculadoramvp/activity/MainActivity.java]: done

**Review by nicolbidone — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**